state/initialize_test.go

```diff
@@ -376,12 +385,12 @@ func (s *InitializeSuite) testBadModelConfig(c *gc.C, update map[string]interfac
 	st.Close()
 	ctlr.Close()
 
-	s.openState(c, st.ModelTag())
+	s.openState(c, names.NewModelTag(st.ModelUUID()))
```

Comment:
So st.ModelUUID() needs to go as it is semantically equivalent to st.ModelTag().
Why not just use s.Model.ModelUUID() here? We have s.Model available